DOCS/man/input.rst

```diff
@@ -554,6 +554,12 @@ Remember to quote string arguments in input.conf (see `Flat command syntax`_).
     because index2 refers to the target entry, not the index the entry
     will have after moving.)
 
+``playlist-order [<index1> [<index2> [...]]``
+    Reorder playlist to start with playlist entry at index1, followed by the
+    entry at index2 and so on. Entries not mentioned in the list are removed.
+    Removing the current entry stops playback and starts playing the next kept
+    entry. Index values start counting with 0.
```

Comment:
If it starts at 0 why is example starting at 1? Also I don't understand the use case for this

Reply:
I've taken the example from `playlist-move` and `run` where _index parameter names_ start counting from 1, even though _index values_ in `playlist-move` is 0 based. But I guess name and value are independent, no?

Say you have (0) b.mkv, (1) notes.txt, (2) a.mp3, (3) c.jpg on your playlist. After `playlist-order 2 0 3` it will be a.mp3, b.mkv, c.jpg.

As the playlist gets bigger you have to use more and more `playlist-move`s and `playlist-remove`s to achieve the desired state, whereas you could use a single `playlist-order`. The latter is way faster and I think is more convenient to use.

I almost always sort the playlist and filter out non-media files.